com.ampl.ide.yaml

```diff
@@ -0,0 +1,39 @@
+app-id: com.ampl.ide
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: amplide.sh
+tags : 'proprietary'
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=host
```

Comment:
Why?

Reply:
x11 and wayland are needed to display the ide. The host permission is needed because this application is not developed with flatpak in mind. This is an IDE for coding.

Reply:
this is broad filesystem permission for a piece of software we can't verify. Defeat the purpose on sandboxing at that point.

Reply:
I can reduce it to xdg-document or something else but unexperienced users will have problems

Reply:
Can this be considered solved?